docs/pages/guides/dom-components.mdx

```diff
@@ -62,26 +62,18 @@ export default function App() {
 
 To pass props to the underlying native **WebView**, add a `dom` object to the component:
```

Comment:
Let's remove this line since it referenced the example code block  belowwhich has already been removed.